### PR TITLE
created major modules: core, data, datafiller, restapi

### PR DESCRIPTION
**Pull request by BolkunetsAlexandr**
created major modules: core, data, datafiller, restapi

### CODE REVIEW

File: core/core.iml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mx510ss) [inline core/core.iml]: We don't need this file - it 'IDE' specific file

File: data/data.iml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mx510ss) [inline data/data.iml]: Don't need this file - please remove and add to gitignore

File: dataFiller/dataFiller.iml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mx510ss) [inline dataFiller/dataFiller.iml]: Dont need this file

File: restAPI/restAPI.iml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mx510ss) [inline restAPI/restAPI.iml]: Dont need this file

File: socmap.iml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mx510ss) [inline socmap.iml]: dont need this file

**Review by mx510ss — CHANGES_REQUESTED** [review]
Please cleanup code - remove all *.iml files

**Comment by BolkunetsAlexandr** [comment]
removed all .iml files

**Review by mx510ss — APPROVED** [review]

**Comment by mx510ss** [comment]
Merged to master

**Pull request merged**